### PR TITLE
Dolly LLM

### PR DESCRIPTION
**Pull request by chandralegend**
- Added Langchain
- Ability to parse context and ask questions
- Ability to create own prompts (Prompt Engineering). (With proper prompting can build chatbots)

### CODE REVIEW

**Comment by marsninja** [comment]
This is an awesome idea @chandralegend !!!

File: jaseci_ai_kit/jac_nlp/jac_nlp/dolly/dolly.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from jaseci.jsorc.live_actions import jaseci_action
+import torch
+from .instruct_pipeline import InstructionTextGenerationPipeline
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+def setup(model: str = "dolly-v2-3b"):
```
Comment (ypkang) [inline jaseci_ai_kit/jac_nlp/jac_nlp/dolly/dolly.py]: setup should be decorated with jaseci_action too.

File: jaseci_ai_kit/jac_nlp/jac_nlp/dolly/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+transformers>=4.28.1
+numpy>=1.23.5
+torch>=2.0.0
+accelerate>=0.18.0
```
Comment (ypkang) [inline jaseci_ai_kit/jac_nlp/jac_nlp/dolly/requirements.txt]: bound the version requirements.

**Comment by ypkang** [comment]
@chandralegend Let's fix the comments i left and merge this. Then we can work on the fine-tuning action.

**Review by ypkang — APPROVED** [review]

**Pull request merged**